lib/ecto/schema.ex

```diff
@@ -398,8 +401,9 @@ defmodule Ecto.Schema do
 
   """
   defmacro field(name, type \\ :string, opts \\ []) do
+    primary_key = Keyword.get(opts, :primary_key, false)
     quote do
-      Ecto.Schema.__field__(__MODULE__, unquote(name), unquote(type), false, unquote(opts))
```

Comment:
I do not see the value,

Default @primary_key 
use explicit call:
Ecto.Schema.__field__(__MODULE__, name, type, true, opts)
https://github.com/elixir-lang/ecto/pull/1210/files#diff-cb8b14078d10148b5f2f3f8635c75892R342

is good this signature for __field__